_docs/never-subclass-components.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Never Subclass Components
+title: Never Subclass Other Components
```

Comment:
"Other" is too vague, IMO. I feel the heading's not really the problem, but the content should be fixed to flow better with the heading.